src/info/url.rs

```diff
@@ -35,7 +36,11 @@ fn get_url(repo: &Repository) -> Result<String> {
     }
 
     let remote_url = match remote_url {
-        Some(url) => url,
+        Some(url) => {
+            let pattern = Regex::new(r"(https?://)([^@]+@)").unwrap();
+            let replaced_url = pattern.replace(&url, "$1").to_string();
+            replaced_url
```

Comment:
You could maybe extract this logic into a separate method with a good name.

It will be easier to unit test it

Reply:
@spenserblack, is it okay to remove everything that you added in that commit?

Reply:
Yes, please do :smile: